github/teams.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2013 The go-github AUTHORS. All rights reserved.
+// Copyright 2018 The go-github AUTHORS. All rights reserved.
```

Comment:
Please revert the copyright date back to the original `2013` value.
Any new files that are added to the repo would get a new `2018` date, but since this file moved, it retains its original date. If people are curious, git can show them the origins (and complete change history) of this file.